packages/beauty-portal-theme/src/components/Footer/index.tsx

```diff
@@ -31,6 +32,8 @@ const Footer: FunctionComponent = () => {
     }
   `);
 
+  console.log(data);
```

Comment:
Can you remove all console logs from any React files?

Reply:
done